This patch fixes some function headers and imports in order for 'make check'
to work.

It was first introduced in version 3.8.3-9
diff -Naur id3lib-3.8.3.orig/examples/findeng.cpp id3lib-3.8.3/examples/findeng.cpp
--- id3lib-3.8.3.orig/examples/findeng.cpp	2003-03-02 01:23:00.000000000 +0100
+++ id3lib-3.8.3/examples/findeng.cpp	2009-12-10 00:58:12.173795997 +0100
@@ -9,7 +9,7 @@
 using std::cout;
 using std::endl;
 
-int main(unsigned argc, char* argv[])
+int main(int argc, char* argv[])
 {
   ID3D_INIT_DOUT();
   ID3D_INIT_WARNING();
diff -Naur id3lib-3.8.3.orig/examples/findstr.cpp id3lib-3.8.3/examples/findstr.cpp
--- id3lib-3.8.3.orig/examples/findstr.cpp	2003-03-02 01:23:00.000000000 +0100
+++ id3lib-3.8.3/examples/findstr.cpp	2009-12-10 00:57:48.036819825 +0100
@@ -9,7 +9,7 @@
 using std::cout;
 using std::endl;
 
-int main(unsigned argc, char* argv[])
+int main(int argc, char* argv[])
 {
   ID3D_INIT_DOUT();
   ID3D_INIT_WARNING();
diff -Naur id3lib-3.8.3.orig/examples/test_io.cpp id3lib-3.8.3/examples/test_io.cpp
--- id3lib-3.8.3.orig/examples/test_io.cpp	2003-03-02 01:23:00.000000000 +0100
+++ id3lib-3.8.3/examples/test_io.cpp	2009-12-10 00:57:03.612111640 +0100
@@ -11,6 +11,9 @@
 #include <id3/io_strings.h>
 #include <id3/utils.h>
 
+using std::cin;
+using std::hex;
+using std::dec;
 using std::cout;
 using std::endl;
 using std::cerr;
@@ -18,7 +21,7 @@
 using namespace dami;
 
 int
-main(size_t argc, const char** argv)
+main(int argc, const char** argv)
 {
   ID3D_INIT_DOUT();
   ID3D_INIT_WARNING();
